cumulus/ansible/tasks/volume.py

```diff
@@ -67,7 +37,11 @@ def attach_volume(profile, cluster, instance, volume, path,
         'girder_volume_id': volume['_id'],
         'girder_cluster_id': cluster['_id'],
         'region': profile['regionName'],
+        'profile_id': profile['_id'],
         'volume_id': volume['ec2']['id'],
+        'volume_name': volume['name'],
+        'volume_size': volume['size'],
+        'volume_zone': volume['zone'],
         'instance_id': instance['instance_id'],
```

Comment:
@cjh1 yeah,  if we want to use the ```default(omit)``` syntax we will probably have to add these variables depending on the state of the volume (e.g. exists/doesn't exist)